src/screens/Inventory/components/InventoryUpdateCard.tsx

```diff
@@ -0,0 +1,59 @@
+import { Card, Typography } from '@material-ui/core';
+import { createStyles, Theme, withStyles } from '@material-ui/core/styles';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import {
+    EMPTY_PRODUCT,
+    selectCurrentSiteInventoryById,
+    selectProductById
+} from '../../../lib/redux/inventoryDataSlice';
+import { RootState } from '../../../lib/redux/store';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    arrow: {
+      color: theme.palette.text.primary,
+    },
+    cardContent: {
+      flex: 1,
+      padding: 16,
+    },
+    cardContainer: {
+      border: `1px solid ${theme.palette.divider}`,
+      boxShadow: 'none',
+      borderRadius: 10,
+      display: 'flex',
+      marginBottom: 10,
+    },
+    arrowSpacing: {
+      padding: 0,
+    },
+  });
+
+interface InventoryUpdateCardProps {
+  classes: { arrow: string; cardContent: string; cardContainer: string; arrowSpacing: string };
+  inventoryId: string;
+  updatedQuantity: number;
+  createdAt: string;
+  userId: string;
+}
+
+function InventoryUpdateCard(props: InventoryUpdateCardProps) {
+  const { classes, inventoryId, updatedQuantity, createdAt, userId } = props;
+  const product =
+    useSelector((state: RootState) =>
+      selectProductById(state, selectCurrentSiteInventoryById(state, inventoryId)?.productId || ''),
+    ) || EMPTY_PRODUCT;
```

Comment:
This same code is in `PurchaseRequest.tsx`. Is there any way to define this selector in `InventoryData` and import it?

Just to clarify for my understanding: The only difference between this and `selectCurrentInventoryProduct` in `inventoryData` is that this function doesn't rely on `currentInventoryId` right?

Reply:
Right. I figured this shouldn't rely on currentInventory in case we need a history feed pulling update cards for different inventory items